plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/TextRecordWriter.java

```diff
@@ -50,6 +51,13 @@ public TextRecordWriter(Path path, JobConf jobConf, Properties properties, boole
         rowSeparator = rowSeparatorByte;
         output = path.getFileSystem(jobConf).create(path, Reporter.NULL);
         compressedOutput = createCompressedStream(jobConf, output, isCompressed);
+
+        Optional<String> skipHeaderLine = Optional.ofNullable(properties.getProperty("skip.header.line.count"));
+        if (skipHeaderLine.isPresent()) {
+            if (Integer.parseInt(skipHeaderLine.get()) == 1 && textHeaderWriter.isPresent()) {
```

Comment:
s/get()/orThrow()

if `Integer.parseInt(skipHeaderLine.get()) == 1 && textHeaderWriter.isEmpty()` here, then we should fail

Reply:
I made a change, tough I am not sure I like how it looks like now